resources/compass/charts/provisioner/values.yaml

```
@@ -10,7 +10,7 @@ database:
   schemaFilePath: "/assets/database/provisioner.sql"
 
 tests:
-  enabled: false
+  enabled: true
```

```suggestion
  enabled: false
```
It is set to true by the overrides in the pipeline.